db/data/regions/XK/en.yml

```diff
@@ -5,5 +5,5 @@ en:
       addresses:
         zip:
           errors:
-            invalid_for_country: Enter a valid postal code for Kosovo or leave it
-              blank
+            invalid_for_country_instructional: Enter a valid postal code for Kosovo
+            invalid_for_country_informative: Postal code doesn't match Kosovo
```

Comment:
Another one that i'm not a huge fan of... maybe `Postal code isn't valid for Kosovo`